deploy/start.sh

```diff
@@ -33,19 +33,8 @@ nginx
 # Start API app
 LISTENING_PORT="8000"
 
-if [ "x$REACT_APP_ENABLE_RBAC" == "x" ]; then
-    echo "RBAC flag not configured, only launch registry app"
-    if [ "x$PURVIEW_NAME" == "x" ]; then
-        echo "Purview flag is not configured, run SQL registry"
-        cd sql-registry
-        uvicorn main:app --host 0.0.0.0 --port $LISTENING_PORT
-    else
-        echo "Purview flag is configured, run Purview registry"
-        cd purview-registry
-        uvicorn main:app --host 0.0.0.0 --port $LISTENING_PORT
-    fi
-else
-    echo "RBAC flag configured, launch both rbac and reigstry apps"
+if [ "$REACT_APP_ENABLE_RBAC" == "true" ]; then
```

Comment:
Can this work if env is set to True

Reply:
No, only works when it's set as "true", according to our provision code 
```
"[if(equals(parameters('enableRBAC'),'Yes'),json('true'),json('null'))]"
```